src/command_modules/azure-cli-hdinsight/HISTORY.rst

```diff
@@ -0,0 +1,9 @@
+.. :changelog:
+
+Release History
+===============
+
+0.0.1
```

Comment:
Begin with `0.1.0`. The last segment of the version is used to part patches.

Reply:
Updated.